builtin/logical/nomad/secret_token.go

```diff
@@ -0,0 +1,59 @@
+package nomad
+
+import (
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+const (
+	SecretTokenType = "token"
+)
+
+func secretToken(b *backend) *framework.Secret {
```

Comment:
Can you attach this to the backend instead of passing it in?

Reply:
Fixed in https://github.com/hashicorp/vault/pull/3401/commits/6560e3c24adf9bdaf56371b0cbfecacd6d19e256